src/OrchardCoreContrib.Gravatar/Views/UserMenu.cshtml

```diff
@@ -32,6 +29,9 @@
                 <li><a class="dropdown-item" asp-area="OrchardCore.Users" asp-action="Display" asp-controller="Admin" asp-route-id="@userId" asp-route-returnUrl="@FullRequestPath"><span><i class="far fa-address-card" aria-hidden="true"></i></span>&nbsp;<span> @T["Profile"]</span></a></li>
             }
             <li><a class="dropdown-item" asp-area="OrchardCore.Users" asp-controller="Account" asp-action="ChangePassword" asp-route-returnUrl="@FullRequestPath"><span><i class="fas fa-key" aria-hidden="true"></i></span>&nbsp;<span> @T["Change password"]</span></a></li>
+            
+            @await RenderSectionAsync("UserMenuItems", required: false)
```

Comment:
Why do we need this?

Reply:
It's something new we added to allow others to inject items in this menu. We should at some point submit a PR to remove most of these menu items from the view and use filters to inject them into the view

Reply:
Happy to hear that, I will have a look and merge your PR ASAP

Reply:
@MikeAlhayek I presume this should be in the Users module, right? after that we can inject the menu items from other modules

Reply:
It should only be in UserMenu view which will optionally allow others to inject into this zone